examples/miscellaneous/miscellaneous_ex12/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#set -x
+
+source $LIBMESH_DIR/examples/run_common.sh
+
+example_name=miscellaneous_ex12
+
+options="-pc_type lu -pc_factor_mat_solver_package mumps"
```

Comment:
According to their doc they do: http://eigen.tuxfamily.org/dox-devel/group__TopicSparseSystems.html.
In this example Eigen classes were helpful for doing lots of small matrix-vector arithmetic.